modules/security_hub/variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "account_id" {
+  type        = string
+  default     = null
+  description = "AWS Account ID"
```

Comment:
What AWS Account ID should be entered here? The description could do with some updating 🙂